steps/appdome-build-2secure-android/1.0.1/step.yml

```diff
@@ -0,0 +1,52 @@
+title: Appdome Build
+summary: |
+  Builds a mobile app using Appdome's platform
+description: |
+  Integration that allows activating security and app protection features, bulding and signing mobile apps using Appdome's API.
```

Comment:
We tend to add more links to the description to for example onboarding pages and so on. So new users who does not yet fully know what Appdome offers can easily check it out. 

This is not mandatory just a nice to have thing.

Reply:
fix done.